app/views/curation_concerns/file_sets/_media_ext.html.erb

```diff
@@ -0,0 +1,27 @@
+  <figure class="image" title="<%= @presenter.alt_text.first %>">
+    <div id="image">
+        <%= link_to image_tag("external_resource_icons/3D_chain.png"), "http://www.google.com", target: "_blank" %>
+    </div>
+    <figcaption><%= presenter.attribute_to_html(:caption, render_as: :markdown, label: '') %></figcaption>
+  </figure> <!-- /.image/media -->
```

Comment:
Add some help text below the caption. It could be:

```
<p>This is an external resource hosted on another website.  <a href="<%= @presenter.whatever the ext URL field is %>" title="<%= @presenter.title.first %>">View object</a> at originating website.</p>
```